deploy/jenkins-ci/Makefile

```diff
@@ -45,13 +57,18 @@ BUMP_BRANCH_ID ?= $(BUILD_TAG)
 
 DOCKER_NAME ?= docker.io/kiali/kiali
 
-ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+ifeq ($(RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE),y)
   IS_SNAPSHOT ?= y
```

Comment:
Is this OK?

I'm wondering because i see that this also release a `v$(BACKEND_VERSION_BRANCH)`
```
	docker tag "$(DOCKER_NAME):v$(BACKEND_VERSION)" "$(DOCKER_NAME):v$(BACKEND_VERSION_BRANCH)"
```

Reply:
I think it's OK. The `v$(BACKEND_VERSION_BRANCH)` should only be pushed when it's an actual release (major/minor/patch) and not a snapshot. Last time this happened was for v0.10:
![image](https://user-images.githubusercontent.com/23639005/52676735-bf6aff80-2ef0-11e9-9667-0e770292d991.png)

The *v0.10* is the "BACKEND_VERSION_BRANCH". It should always point to the latest patch release of a minor stream. When minor v0.10.0 was released, the tag v0.10 was also created and it was an alias of v0.10.0. When patch v0.10.1 was released, the tag v0.10 was updated to point to that patch release.